src/get-started/quickstart.md

```diff
@@ -73,6 +73,12 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
      -Dexec.args="--inputFile=pom.xml --output=counts"
 ```
 
+{:.runner-apex}
+``` 
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```

Comment:
Could we teach the users here how to run on Apex locally and on a remote cluster?